doc/user/bgp.rst

```diff
@@ -1045,6 +1045,15 @@ Configuring Peers
 .. index:: neighbor PEER port PORT
 .. clicmd:: neighbor PEER port PORT
 
+.. index:: [no] neighbor PEER password PASSWORD
+.. clicmd:: [no] neighbor PEER password PASSWORD
+
+   Set a MD5 password to be used with the tcp socket that is being used
+   to connect to the remote peer.  Please note if you are using this
+   command with a large number of peers on linux you should consider
+   modifying the `net.core.optmem_max` sysctl to a larger value to
```

Comment:
double backticks?

Reply:
I'm not sure what you are asking, can you elaborate?

Reply:
Shouldn’t a single backtick replaced to double backticks? I mean for syntax highlighting.

Reply:
@ton31337, nah, single is enough.

Reply:
👌